### DISCUSSION TITLE
How are royalties distributed for bundle sales?

### DISCUSSION THREAD

**Discussion by levidyrek** [ProjectOpenSea/seaport#586] [Q&A]
Let's say I have an NFT from Collection A and Collection B and sell them as a bundle for 1 ETH. Collection A has a 10% royalty and Collection B has 5%. How much does each creator get?

**Comment by howardpen9** [comment]
Similar question but for ERC1155: What if I had a collection that has three tokenID (ERC1155 standard), tokenId=0 with 10% Royalties Fee, tokenId=1 with 5% Royalties Fee.  Can we still get the royalties fee successfully on OpenSea?
